admin/app/controllers/DeploysRadiatorController.scala

```diff
@@ -16,8 +16,12 @@ trait DeploysRadiatorController extends Controller with Logging with Requests {
   val riffRaff: RiffRaffService
   val teamcity: TeamcityService
 
-  def getDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String]) = Action.async {
-    riffRaff.getRiffRaffDeploys(pageSize, projectName, stage).map(ApiResults(_))
+  def getCompletedDeploys(pageSize: Option[String], projectName: Option[String], stage: Option[String]) = Action.async {
```

Comment:
Is there much point in keeping `projectName` as an argument? Could hardcode to dotcom:all